raiden_contracts/deploy/__main__.py

```diff
@@ -101,7 +101,10 @@ def deploy(
 
         # Get tx receipt to get contract address
         log.debug("Deploying %s txHash=%s" % (contract_name, encode_hex(txhash)))
-        receipt = check_succesful_tx(self.web3, txhash, self.wait)
+        (receipt, tx) = check_succesful_tx(self.web3, txhash, self.wait)
+        if not receipt['contractAddress']:  # happens with Parity
+            receipt = dict(receipt)
+            receipt['contractAddress'] = tx['creates']
```

Comment:
Just a note: weirdly, `creates` is not documented properly. I only found it mentioned in https://web3py.readthedocs.io/en/stable/examples.html?highlight=creates#looking-up-transactions.